pkg/util/util.go

```diff
@@ -0,0 +1,22 @@
+package util
+
+import (
+	"bufio"
+	"os"
+)
+
+// ReadLines reads the content of the given file into a string slice
+func ReadLines(fileName string) ([]string, error) {
```

Comment:
This appears to be a copy of a function only used by builders.  The rule for library go is that it must be of reasonable complexity (this may fail that) and must be used by multiple repos (this does fail that).